TYPO3.Flow/Classes/TYPO3/Flow/Utility/Lock/FlockLockStrategy.php

```diff
@@ -99,6 +99,7 @@ public function release() {
 				$success = FALSE;
 			}
 			fclose($this->filePointer);
+			Files::unlink($this->lockFileName);
```

Comment:
This creates a small race condition, for when a second process has just opened/created the file and not yet got a lock on it (being in between Ln 74 and 79). The file would then possibly be deleted before the other process gets ahold of the lock and hence fails.